sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/AppendBlobAsyncClient.java

```diff
@@ -50,12 +52,12 @@
     /**
      * Indicates the maximum number of bytes that can be sent in a call to appendBlock.
      */
-    public static final int MAX_APPEND_BLOCK_BYTES = 4 * Constants.MB;
+    public final static int MAX_APPEND_BLOCK_BYTES = 4 * Constants.MB;
```

Comment:
Did the Checkstyle rule change to order `final static` instead of `static final` as it has been?

Reply:
nope, fixed!